### PR TITLE
Update web-api-help-pages-using-swagger.md

### PR DESCRIPTION
**Pull request by FinHorsley**
### What's this?

Hopefully this helps make the Swagger/OpenAPI definition a bit more clear. Whilst using Swagger I've found the terms confusing as they're often used interchangeably (e.g. Swashbuckle uses `Swagger`, whilst NSwag refers to it as `OpenAPI`)

### CODE REVIEW

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 1]
```diff
@@ -8,25 +8,29 @@ ms.date: 07/06/2020
...
 
-In this article, the [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag) .NET Swagger implementations are showcased:
+The two main OpenAPI implementations for .Net are [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag), see:
 
-* **Swashbuckle.AspNetCore** is an open source project for generating Swagger documents for ASP.NET Core Web APIs.
+* [Getting Started with Swashbuckle](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-3.1)
+* [Getting Started with NSwag](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-nswag?view=aspnetcore-3.1)
```
Comment (FinHorsley) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:20]: Not sure about the links here, should they be relative?

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 2]
```diff
@@ -8,25 +8,29 @@ ms.date: 07/06/2020
...
+Swagger (OpenAPI) is a language-agnostic specification for describing REST APIs. It allows both computers and humans to understand the capabilities of an API service without any direct access to the implementation (source code, network access, documentation). Its main goals are to help minimise the amount of work needed to connect decoupled services, as well as reduce the amount of time needed to accurately document a service.
 
-In this article, the [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag) .NET Swagger implementations are showcased:
+The two main OpenAPI implementations for .Net are [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag), see:
 
-* **Swashbuckle.AspNetCore** is an open source project for generating Swagger documents for ASP.NET Core Web APIs.
+* [Getting Started with Swashbuckle](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-3.1)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:19]:
```suggestion
* [Getting Started with Swashbuckle](xref:aspnet/core/tutorials/getting-started-with-swashbuckle)
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:20]:
```suggestion
* [Getting Started with NSwag](xref:tutorials/getting-started-with-nswag)
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 3]
```diff
@@ -8,25 +8,29 @@ ms.date: 07/06/2020
...
+Swagger (OpenAPI) is a language-agnostic specification for describing REST APIs. It allows both computers and humans to understand the capabilities of an API service without any direct access to the implementation (source code, network access, documentation). Its main goals are to help minimise the amount of work needed to connect decoupled services, as well as reduce the amount of time needed to accurately document a service.
 
-In this article, the [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag) .NET Swagger implementations are showcased:
+The two main OpenAPI implementations for .Net are [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag), see:
 
-* **Swashbuckle.AspNetCore** is an open source project for generating Swagger documents for ASP.NET Core Web APIs.
+* [Getting Started with Swashbuckle](xref:aspnet/core/tutorials/getting-started-with-swashbuckle)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:19]:
```suggestion
* [Getting Started with Swashbuckle](tutorials/getting-started-with-swashbuckle)
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 4]
```diff
@@ -8,25 +8,29 @@ ms.date: 07/06/2020
...
+* [Getting Started with NSwag](xref:tutorials/getting-started-with-nswag)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:20]:
```suggestion
* [Getting Started with NSwag](xref:tutorials/get-started-with-nswag)
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger.md [hunk 5]
```diff
@@ -8,25 +8,29 @@ ms.date: 07/06/2020
...
+Swagger (OpenAPI) is a language-agnostic specification for describing REST APIs. It allows both computers and humans to understand the capabilities of an API service without any direct access to the implementation (source code, network access, documentation). Its main goals are to help minimise the amount of work needed to connect decoupled services, as well as reduce the amount of time needed to accurately document a service.
 
-In this article, the [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag) .NET Swagger implementations are showcased:
+The two main OpenAPI implementations for .Net are [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and [NSwag](https://github.com/RicoSuter/NSwag), see:
 
-* **Swashbuckle.AspNetCore** is an open source project for generating Swagger documents for ASP.NET Core Web APIs.
+* [Getting Started with Swashbuckle](xref:tutorials/getting-started-with-swashbuckle)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger.md:19]:
```suggestion
* [Getting Started with Swashbuckle](xref:tutorials/get-started-with-swashbuckle)
```

**Review by scottaddie — APPROVED** [review]

**Comment by scottaddie** [comment]
Thank you for your help, @FinHorsley!

**Pull request merged**